### PR TITLE
Add Trino 402 release notes

### LINKED ISSUE #14780: Release notes for 402

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 402 release. Fixes #14780 

Also removing one incorrect release note from 399, as due to a bug, the feature mentioned was not functional.

## Non-technical explanation

N/A

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

## Verification for each pull request

Format: maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 26 October 2022

@ebyhr #14766 ✅ rn ✅ docs
@ebyhr #14590 ✅ rn ✅ docs
@ebyhr #14782 ✅ rn ✅ docs
@ebyhr #14336 ✅ rn ❌ docs
@ebyhr #14784 ✅ rn ✅ docs

## 27 October 2022 ✅ 

@kokosing #14762 ✅ rn ✅ docs
@sopel39 #14781 ✅ rn ✅ docs
@arhimondr #14785 ✅ rn ✅ docs
@martint #14773 ✅ rn ✅ docs
@findepi #14763 ✅ rn ✅ docs
@arhimondr #14794 ✅ rn ✅ docs
@losipiuk #14756 ✅ rn ✅ docs
@ebyhr #14788 ✅ rn ✅ docs
@raunaqmorarka #14777 ✅ rn ✅ docs
@kasiafi #14800 ✅ rn ✅ docs

## 28 October 2022

@losipiuk #14799 ✅ rn ✅ docs
@ebyhr #13800 ✅ rn ✅ docs
@sopel39 #14802 ✅ rn ✅ docs
@kokosing #14770 ✅ rn ✅ docs
@kokosing #14645 ✅ rn ✅ docs
@losipiuk #8962 ❌ rn ✅ docs

## 29 October 2022 ✅ 

@losipiuk #14806 ✅ rn ✅ docs

## 31 October 2022 ✅ 

@sopel39 #14815 ✅ rn ✅ docs
@hashhar #14805 ✅ rn ✅ docs
@losipiuk #14819 ✅ rn ✅ docs
@hashhar #14570 ✅ rn ✅ docs
@kokosing #14749 ✅ rn ✅ docs
@sopel39 #14838 ✅ rn ✅ docs
@ebyhr #14813 ✅ rn ✅ docs
@ebyhr #14795 ✅ rn ✅ docs
@hashhar #14809 ✅ rn ✅ docs
@hashhar #14139 ✅ rn ✅ docs

## 1 November 2022 ✅ 

@hashhar #14839 ✅ rn ✅ docs
@electrum #14857 ✅ rn ✅ docs
@martint #14528 ✅ rn ✅ docs
@electrum #14851 ✅ rn ✅ docs
@hashhar #14855 ✅ rn ✅ docs

## 2 November 2022

@kokosing #14653 ✅ rn ➖ docs (#14870)
@sopel39 #14627 ✅ rn ✅ docs
@sopel39 #14823 ✅ rn ✅ docs
@hashhar #14810 ✅ rn ✅ docs
@sopel39 #14866 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-402.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## MongoDB connector
+
+* Support predicate pushdown on `date`, `time(3)`, `timestamp(3)` and
+  `timestamp(3) with time zone` columns. ({issue}`14795`)
+
+## SQL Server connector
+
+* Improve performance of certain queries which use the `OR` operator. ({issue}`14570`)
+* Improve performance of queries with predicates involving the `nullif` function
+  or arithmetic expressions. ({issue}`14570`)
```
Comment (colebow) [inline docs/src/main/sphinx/release/release-402.md:60]: @hashhar does this look ok? I cut down what was proposed pretty significantly - if users are curious, they generally can look at the PR to get additional context.

File: docs/src/main/sphinx/release/release-399.md [hunk 2]
```diff
@@ -58,11 +58,6 @@
   with other Iceberg catalog implementations. ({issue}`14384`)
 * Allow setting the AWS STS endpoint and region when using a Glue metastore. ({issue}`14412`)
 
-## MongoDB connector
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-399.md:61]: Was that wrong?

File: docs/src/main/sphinx/release/release-402.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# Release 402 (2 Nov 2022)
+
+## General
+
+* Fix query hanging indefinitely when [fault-tolerant execution](/admin/fault-tolerant-execution)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-402.md:5]: query processing

File: docs/src/main/sphinx/release/release-402.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  is enabled and a [stage](trino-concept-stage) of the query produces no data. ({issue}`14794`)
+
+## Blackhole connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-402.md:10]: Hahah .. thats kinda funny

File: docs/src/main/sphinx/release/release-402.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+## Clickhouse connector
+
+* Reuse an existing JDBC connection for metadata queries. This can be disabled
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-402.md:14]: So is it enabled by default .. right?

File: docs/src/main/sphinx/release/release-402.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  with the `query.reuse-connection` configuration property. ({issue}`14653`)
+
+## Delta Lake connector
+
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-402.md:19]: Are they removed from the docs?

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-399.md (hunk 2 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-399.md:61]: Yeah, see #14795

File: docs/src/main/sphinx/release/release-402.md (hunk 6 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-402.md:19]: Yeah, I think they were removed when they were deprecated.

File: docs/src/main/sphinx/release/release-402.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+## Hudi connector
+
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+
+## Iceberg connector
+
+* Add support for skipping archiving when committing to a table in the Glue
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/release/release-402.md]: @colebow .. I think we should avoid triple "ing" ..

File: docs/src/main/sphinx/release/release-402.md (hunk 4 above)
Comment (ebyhr) [reply docs/src/main/sphinx/release/release-402.md:10]: Remove mention for materialized view.

File: docs/src/main/sphinx/release/release-402.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+* Fix memory leak and improve memory tracking during large `INSERT` queries. ({issue}`14823`)
+
+## Druid connector
+
+* Reuse JDBC connections for metadata queries. This can be disabled with the
+  `query.reuse-connection` configuration property. ({issue}`14653`)
+
+## Hive connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
```
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-402.md:32]: Remove mention for materialized view.

File: docs/src/main/sphinx/release/release-402.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+
+## Iceberg connector
+
+* Add support to skip archiving when committing to a table in the Glue
+  metastore and the `iceberg.glue.skip-archive` configuration property is set
+  to true. ({issue}`13413`)
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
```
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-402.md:51]: Remove mention for materialized view.

File: docs/src/main/sphinx/release/release-402.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+* Fix memory leak and improve memory tracking during large `INSERT` queries. ({issue}`14823`)
+
+## MariaDB connector
+
+* Reuse JDBC connections for metadata queries. This can be disabled with the
+  `query.reuse-connection` configuration property. ({issue}`14653`)
+
+## Memory connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
```
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-402.md:67]: Remove mention for materialized view.

**Pull request merged**